__tests__/server.test.js

```diff
@@ -68,36 +68,109 @@ describe("/api/articles", () => {
 });
 
 describe("/api/articles/:article_id", () => {
-  test("GET:200 responds with article by its id", () => {
-    return request(app)
-      .get("/api/articles/1")
-      .expect(200)
-      .then(({ body: { article } }) => {
-        expect(typeof article.author).toBe("string");
-        expect(typeof article.title).toBe("string");
-        expect(article.article_id).toBe(1);
-        expect(typeof article.body).toBe("string");
-        expect(typeof article.topic).toBe("string");
-        expect(typeof article.created_at).toBe("string");
-        expect(typeof article.votes).toBe("number");
-        expect(typeof article.article_img_url).toBe("string");
-      });
-  });
-  test("GET:404 responds article ID not found", () => {
-    return request(app)
-      .get("/api/articles/999")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toBe("Article ID does not exist!");
-      });
+  describe("GET", () => {
+    test("GET:200 responds with article by its id", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(article.article_id).toBe(1);
+          expect(typeof article.body).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+        });
+    });
```

Comment:
Are you sure that the other article information is related to article 1? You have access to the test data, so make your test more specific to the requested article